setup.py

```diff
@@ -174,15 +174,20 @@ def finalize_options(self):
 
     def run(self):
         import os
-        os.system("find . -name '*.pyc' | xargs rm -f")
-        os.system("rm -f python-build-stamp-2.4")
-        os.system("rm -f MANIFEST")
-        os.system("rm -rf build")
-        os.system("rm -rf dist")
-        os.system("rm -rf doc/_build")
-        os.system("rm -f sample.tex")
-
-
+        import shutil
+        for root, dirs, files in os.walk("."):
```

Comment:
This should not look in the current working directory, but in the directory that `setup.py` resides in. You can use the `__file__` variable to get at that.